src/commands/deploy/setPreDeployTaskForDotnet.ts

```diff
@@ -33,6 +33,16 @@ export async function setPreDeployTaskForDotnet(context: IDeployContext): Promis
         return;
     }
 
+    // if the user has scmDoBuild configured in a ".deployment" file - assume they've already configured their project's deploy settings
+    const deploymentFilePath: string = path.join(context.effectiveDeployFsPath, constants.deploymentFileName);
```

Comment:
I'm of the opinion that if a user has a deployment file at all, there are custom build scripts that they could have and we may want to avoid setting any preDeployTasks/deploySubpath since it could break those users not just if they have scmDoBuild set.

Reply:
I can jive with that